tools/introspection/source/introspection_app.cpp

```diff
@@ -332,31 +332,31 @@ void IntrospectionApp::printPortIntrospectionData(const std::vector<ComposedPubl
     wprintw(pad, "----------------------------------------------------------------------------\n");
 
     bool needsLineBreak{false};
-    uint32_t currentLine{0};
-    auto printEntry = [&](std::uint32_t maxSize, const std::string& data) -> std::string {
+    uint32_t currentLine{0U};
+    auto printEntry = [&](int32_t maxSize, const std::string& data) -> std::string {
```

Comment:
since `maxSize` is most of the time casted to `size_t`, shouldn't this be `size_t`. Ignore this comment if it makes the code on the caller site uglier.

Reply:
Yes it makes it uglier and one time the paramter comes as negative value. I let it be.